sparse/usr/share/csd/settings.d/10-hw-settings.ini

```diff
@@ -36,6 +36,7 @@ UsbCharging=1
 UsbOtg=1
 Vibrator=1
 Wifi=1
+BluetoothTethering=1
```

Comment:
Nitpick: The existing features are in alphabetical order (except for one other issue in the list)